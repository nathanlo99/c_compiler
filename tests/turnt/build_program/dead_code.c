
int wain(int arg1, int arg2) {
  int a = 4;
  int b = 2;
  int c = 1;
  int d = 0;
  int e = 0;
  d = a + b;
  e = c + d;
  return d;
}
