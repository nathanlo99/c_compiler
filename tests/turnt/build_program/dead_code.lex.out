INT (int) at [2:1 - 2:3]
WAIN (wain) at [2:5 - 2:8]
LPAREN (() at [2:9 - 2:9]
INT (int) at [2:10 - 2:12]
ID (arg1) at [2:14 - 2:17]
COMMA (,) at [2:18 - 2:18]
INT (int) at [2:20 - 2:22]
ID (arg2) at [2:24 - 2:27]
RPAREN ()) at [2:28 - 2:28]
LBRACE ({) at [2:30 - 2:30]
INT (int) at [3:3 - 3:5]
ID (a) at [3:7 - 3:7]
BECOMES (=) at [3:9 - 3:9]
NUM (4) at [3:11 - 3:11]
SEMI (;) at [3:12 - 3:12]
INT (int) at [4:3 - 4:5]
ID (b) at [4:7 - 4:7]
BECOMES (=) at [4:9 - 4:9]
NUM (2) at [4:11 - 4:11]
SEMI (;) at [4:12 - 4:12]
INT (int) at [5:3 - 5:5]
ID (c) at [5:7 - 5:7]
BECOMES (=) at [5:9 - 5:9]
NUM (1) at [5:11 - 5:11]
SEMI (;) at [5:12 - 5:12]
INT (int) at [6:3 - 6:5]
ID (d) at [6:7 - 6:7]
BECOMES (=) at [6:9 - 6:9]
NUM (0) at [6:11 - 6:11]
SEMI (;) at [6:12 - 6:12]
INT (int) at [7:3 - 7:5]
ID (e) at [7:7 - 7:7]
BECOMES (=) at [7:9 - 7:9]
NUM (0) at [7:11 - 7:11]
SEMI (;) at [7:12 - 7:12]
ID (d) at [8:3 - 8:3]
BECOMES (=) at [8:5 - 8:5]
ID (a) at [8:7 - 8:7]
PLUS (+) at [8:9 - 8:9]
ID (b) at [8:11 - 8:11]
SEMI (;) at [8:12 - 8:12]
ID (e) at [9:3 - 9:3]
BECOMES (=) at [9:5 - 9:5]
ID (c) at [9:7 - 9:7]
PLUS (+) at [9:9 - 9:9]
ID (d) at [9:11 - 9:11]
SEMI (;) at [9:12 - 9:12]
RETURN (return) at [10:3 - 10:8]
ID (d) at [10:10 - 10:10]
SEMI (;) at [10:11 - 10:11]
RBRACE (}) at [11:1 - 11:1]
